applications/dashboard/js/src/modal.dashboard.js

```diff
@@ -22,9 +22,8 @@ var DashboardModal = (function() {
 
         defaultSettings: {
             httpmethod: 'get',
-            afterSuccess: function(json, sender) {
-                // Called after the confirm url has been loaded via ajax
-            },
+            afterSuccess: function(json, sender) {},
```

Comment:
```js
$.noop, // function(json, sender)
```

Save a useless function declaration. It is no big deal though. Just something to keep in mind as a good practice.